src/databases/functions/users/user-set-karma-score.js

```diff
@@ -1,10 +1,34 @@
 const {User} = require('../../models');
+const {QueryTypes} = require('sequelize');
+const db = require('../../models/');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You're importing `sequelize` and `User` model from `../../models` twice. It's unnecessary and can be optimized by combining them into a single import statement.
```diff
- const {User} = require('../../models');
- const {QueryTypes} = require('sequelize');
- const db = require('../../models/');
+ const {User, sequelize, QueryTypes} = require('../../models');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->